### ISSUE TITLE
custom image on login screen

### ISSUE THREAD

**Issue by ChhatralaNirmal** [tchiotludo/akhq#901]
Is there any option to change custom image on login page and after the login ?

**Comment by tchiotludo** [comment]
replacing the logo you mean ?

**Comment by ChhatralaNirmal** [comment]
Yes

**Comment by tchiotludo** [comment]
Nope for now, PR are welcome

**Comment by ChhatralaNirmal** [comment]
We would like to replace akhq.io logo with our custom logo

**Comment by jheinitz** [comment]
Good idea. Maybe not simply replacing the existing logo, but add a second logo for branding. Thx! Jens

**Comment by AncutaIoan** [comment]
Hi everyone! 

I'd love to take a shot at this if you're open to it. I've been using AKHQ at a couple of the companies I've worked with, and it would be great to contribute something back as a small token of appreciation. Let me know if you're okay with me picking this up!

Thanks!

**Comment by AncutaIoan** [comment]
Just let me know how would you like this to be done (via /me endpoint, .env directly within ui/etc)
